.github/workflows/commitlint.yml

```diff
@@ -0,0 +1,32 @@
+name: CI
+
+on:
+  pull_request:
+    branches:
+      - main
+jobs:
+  commitlint:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+      - name: Install required dependencies
+        run: |
+          sudo apt install -y git curl
+          curl -sL https://deb.nodesource.com/setup_14.x | sudo -E bash -
+          sudo DEBIAN_FRONTEND=noninteractive apt install -y nodejs
+      - name: Print versions
+        run: |
+          node --version
+          npm --version
+          npx commitlint --version
+      - name: Install commitlint
+        run: |
+          npm install -g @commitlint/cli @commitlint/config-conventional
+      - name: Configure commitlint
+        run: |
+          echo "module.exports = {extends: ['@commitlint/config-conventional']}" > commitlint.config.js
```

Comment:
As noted in our team discussion, this should ideally be either in our regular image that we also use in concourse, or a separate minimal image that contains commitlint. 

Fetching and installing these packages every time would only be reasonable if those were dependencies that are vital to the build or resulting output of the build, so we could ensure that the build would still work. 

With commit linting we only care that it's actually checking the commit message. If that were to break with a newer version, we could still stay on the old and adapt in time. It would not be a reproducibility blocker.

Reply:
Agreed on using a node base image and eventually switching to a prepared commitlint docker image, when upstream provides one.